lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -239,20 +239,30 @@ def stub_email_alert_api_refuses_to_create_subscription(subscriber_list_id, addr
         ).to_return(status: 422)
       end
 
-      def stub_email_alert_api_creates_an_auth_token(subscriber_id, address)
+      def stub_email_alert_api_sends_subscription_verification_email(subscriber_id, address, topic_id)
```

Comment:
Comparing with the adapter, I think we need to replace `subscriber_id` -> `frequency`.

Reply:
I think I introduced this one 😬